demo/index.html

```diff
@@ -124,6 +124,7 @@ <h1>Vbrick SDK Embed Test</h1>
 				<div>
 					<input type="button" name="Play" value="Play" onclick="vbrickEmbed.play()" />
 					<input type="button" name="Pause" value="Pause" onclick="vbrickEmbed.pause()" />
+					<input type="button" name="SetVolume" value="Set Volume to 50%" onclick="vbrickEmbed.setVolume(0.5)" />
```

Comment:
this could be a slider:
something like
```
  <input type="range" min="0" max="1" value="0.1" step="0.01" onchange=
````

Reply:
I'll try that on future updates